bolt-modules/dir/lib/puppet/functions/dir/entries.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# Enumerate files in a given directory
```

Comment:
This should have an example, similar to the other functions. Since the `dirname` can be an absolute path or Puppet file path, I think showing examples for each would be helpful.